pkg/cmd/roachtest/tests/asyncpg_blocklist.go

```diff
@@ -23,7 +23,105 @@ var asyncpgBlocklists = blocklistsForVersion{
 // Please keep these lists alphabetized for easy diffing.
 // After a failed run, an updated version of this blocklist should be available
 // in the test log.
-var asyncpgBlocklist22_1 = asyncpgBlocklist21_2
```

Comment:
flaky tests should be in the `ignoreList`, while tests that fail reliably should be in the `blockList`

right now looks like there's some overlap between the two lists?

Reply:
Oops this should all be ignoreList, fixed.